pkgs/applications/science/misc/openmvs/default.nix

```diff
@@ -57,8 +56,7 @@ stdenv.mkDerivation {
     homepage = "http://cdcseacave.github.io/openMVS/";
     license = licenses.agpl3;
     platforms = platforms.linux;
-    maintainers = with maintainers; [ mdaiter ];
+    maintainers = with maintainers; [ mdaiter timput ];
     # 20190414-174115: CMake cannot find CGAL which is passed as build input
```

Comment:
Is the comment still relevant if this is not broken anymore?

Also, I can verify that this builds (which I did), but I don't really know what this does, so I can't really test it.

Seeing as you're adding yourself as maintainer, I'd trust you that this actually does what it's supposed to.

Reply:
You are correct about that comment, I've removed it now.

I've also changed the commit message to better align with CONTRIBUTING.md